grunt_tasks/webpack.config.js

```diff
@@ -50,6 +53,40 @@ function urlLoader(options) {
     return loader;
 }
 
+function defaultLoaderPlugins() {
+    // Computes a list of plugin paths that are opting to use Girder's loader
+    // configuration directly; these paths will be added to the "include"
+    // directive for the loaders in the config template below.
+    var dirs = fs.readdirSync('plugins');
+    var paths = [];
+
+    dirs.forEach(function (dir) {
+        // Find a plugin config file.
+        var pluginFile = path.resolve('plugins', dir, 'plugin.yml');
+        if (!fs.existsSync(pluginFile)) {
+            pluginFile = path.resolve('plugins', dir, 'plugin.json');
+            if (!fs.existsSync(pluginFile)) {
+                pluginFile = null;
+            }
+        }
```

Comment:
We should try to unify this with the logic inside plugin.js. We now require the list of plugins to be explicitly passed, and only do logic like this if `--all-plugins` is passed.

The easiest thing might be to move this logic out of here, and have some logic inside plugin.js that augments the  paths for all the loaders.